### PR TITLE
fix(GiniBankSDK): Fix error label sizes

### PR DESCRIPTION
**Pull request by vizaknaidavid**
Simulator Screen Shot - iPhone 13 Pro - 2023-04-14 at 15 43 35
Simulator Screen Shot - iPhone 13 Pro - 2023-04-14 at 15 48 24

PIA-3965

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift [hunk 1]
```diff
@@ -46,6 +47,7 @@ final class EditLineItemView: UIView {
                                   for: .normal)
         button.addTarget(self, action: #selector(didTapSave), for: .touchUpInside)
         button.setTitleColor(.GiniBank.accent1, for: .normal)
+        button.titleLabel?.textColor = .GiniBank.accent1
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift:50]: [Clarification] I don't think we need this change.
We have set button.titleLabel?.textColor in line 49

**Review by zladzeyka — COMMENTED** [review]
Great @vizaknaidavid 
Size of the error messages looks good.
I've left 2 small comments. PS I've tested without them and color is still applied.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift [hunk 2]
```diff
@@ -20,6 +20,7 @@ final class EditLineItemView: UIView {
         button.setTitle(title, for: .normal)
         button.addTarget(self, action: #selector(didTapCancel), for: .touchUpInside)
         button.setTitleColor(.GiniBank.accent1, for: .normal)
+        button.titleLabel?.textColor = .GiniBank.accent1
```
[the remark this answers is not in the document]
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift:23]: on iOS 14+ it works with that, but on 13 the text color was black without this line. :(

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift (hunk 1 above)
Comment (vizaknaidavid) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift:50]: Same

**Review by zladzeyka — APPROVED** [review]

**Pull request merged**